README.md

```diff
@@ -63,58 +63,54 @@ jobs:
 ```
 
 <!-- action-docs-inputs action="action.yml" -->
-
 ## Inputs
 
-| name                                 | description                                                                                                                                                                                               | required | default             |
```

Comment:
Please back out the actual content changes here. I'm planning to keep it touched-up by `prettier`, otherwise Restyled will keep trying to put it back -- also, I like the alignment when it's automatically maintained. You can use `npm run readme` to do the [re]generation and re-formatting for you in one go.

Reply:
At that time, I got these changes after running `npm run readme` :see_no_evil:. Probably, a culprit is one of my VS Code extensions.